### ISSUE TITLE
Fix failing v2-beta term details test

### ISSUE THREAD

**Issue by johnbradley** [phenoscape/rphenoscape#211]
A term details test is failing when using the [v2-beta kb.phenoscape API](https://dev.phenoscape.org/api/v2-beta)
```
Failure (test-pk.R:25:3): Test term details
length(unique(g$taxon.label)) is not strictly more than 2. Difference: 0
```
https://github.com/phenoscape/rphenoscape/blob/82e16d96e3ed34c6ddd59ee640ce58310d684c62/tests/testthat/test-pk.R#L25
This error is occurring because `gene_info("socs5")` is no longer returning the `taxon.label=Xenopus` item when using the v2-beta API. The production kb.phenoscape API returns 4 items and the v2-beta kb.phenoscape API returns 3 items. 
This data comes from the `api/gene/search?text=socs5` kb.phenoscape API.
The JSON returned for this item in the production API is below:
```
...
    {
      "@id": "http://xenbase.org/XB-GENEPAGE-479592",
      "label": "socs5",
      "taxon": {
        "@id": "http://purl.obolibrary.org/obo/NCBITaxon_8353",
        "label": "Xenopus"
      },
      "matchType": "exact"
    },
...
```

**Comment by hlapp** [comment]
@balhoff or @Shalsh23 can either of you comment on this being expected, i.e., we are for some reason and on purpose no longer importing Xenopus genes?

**Comment by balhoff** [comment]
It is expected. We only had a few Xenopus phenotype annotations, and we aren't getting any from Monarch right now. Although, I should check the current status of what is available from Monarch because there has been work happening recently on the new Xenopus phenotype ontology.

**Comment by johnbradley** [comment]
Fixed by PR #221
